lightkurve/seismology/core.py

```diff
@@ -602,11 +602,24 @@ def estimate_radius(self, teff, numax=None, deltanu=None):
         """
         numax = self._validate_numax(numax)
         deltanu = self._validate_deltanu(deltanu)
+        if teff == None:
+            teff = self.periodogram.meta.get('teff')
+            if teff == None:
+                raise ValueError("No effective temperature in metadata."
+                                "Please include effective temperature in function call.")
```

Comment:
I wonder if this error message can be more specific about the argument names, for example:
```
You must provide an effective temperature argument (`teff`) to `estimate_radius`,
because the Periodogram object does not contain it in its meta data (i.e. `pg.meta['teff']` is missing).
```